modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -101,7 +101,16 @@ If any of these conditions are true, do not force wipe the disk. Instead, you mu
 
 |`deviceClasses.thinPoolConfig`
 |`object`
-|Contains the configuration to create a thin pool in the LVM volume group.
+|Contains the configuration to create a thin pool in the LVM volume group. 
+
+If you exclude this field, logical volumes are thick-provisioned. 
+
+The limitations of using thick-provisioned storage are as follows:
```

Comment:
```suggestion
The limitations of using thick-provisioned storage include:
```
Just a suggestion for added simplicity.

Reply:
Looking at the [IBM Style's](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording) rules on lead-in sentences for lists, I think that the right answer might be in the middle.
```suggestion
Using thick-provisioned storage includes the following limitations:
```